tmt/result.py

```diff
@@ -153,7 +155,7 @@ def __init__(
                     ResultOutcome.PASS: ResultOutcome.FAIL
                     }.get(self.result, self.result)
             elif ResultInterpret.is_result_outcome(interpret):
-                self.result = ResultOutcome(interpret.value)
+                self.result = ResultOutcome.from_spec(interpret.value)
```

Comment:
This should not need `from_spec` - `interpret` is already an enum, therefore there's no way direct `ResultOutcome(...)` could lead to an error. If `interpret` has been read from a file, `from_spec()` should have been used at that place, and it's no necessary here. We trust our enums :)